orttraining/orttraining/training_ops/cuda/loss/softmaxcrossentropy_impl.cu

```diff
@@ -101,7 +101,12 @@ __global__ void _SparseSoftmaxCrossEntropy(
     CUDA_LONG D) {
   CALCULATE_ELEMENTWISE_INDEX_OR_EXIT(i, N);
   CUDA_KERNEL_ASSERT(label_data[i] >= 0 && label_data[i] < D);
-  output_data[i] = -log_prob_data[i * D + label_data[i]] / (*normalize_factor_data);
+  
+  if(*normalize_factor_data == 0){
```

Comment:
formatting looks off. please turn on auto formatting and see other code for reference.